cli/flox/doc/flox-activate.md

```diff
@@ -10,99 +10,103 @@ flox-activate - activate environments
+This means that for each shell hook various environment variables such as
+`PATH`, `MANPATH`, `PKG_CONFIG_PATH`, `PYTHONPATH`, etc,
+are set to the appropriate values for the environment in which the shell
+hook was defined.
 
-```{.include}
-./include/general-options.md
-./include/environment-options.md
-```
+# OPTIONS
 
 ## Activate Options
 
-[ -- `<command>` [ `<argument>` ] ]
+`-- <command> [ <arguments> ]`
 :   Command to run in the environment.
-    Spawns the command in a subshell
-    that does not leak into the calling process.
+    Spawns the command in a subshell that does not leak into the calling
+    process.
+
+`-t`, `--trust`
+:   Trust a remote environment for this activation.
+    Activating an environment executes a shell hookwhich may execute arbitrary
+    code.
+    This presents a security risk,
+    so you will be prompted whether to trust the environment.
+    Environments owned by the current user are always trusted.
+    You may set certain environments to always be trusted using the config key
+    `trusted_environments."<owner/name>"`,
+    or via the following command:
+    `flox config --set trusted_environments.\"<owner/name>\" trust`.
 
+```{.include}
+./include/environment-options.md
+./include/general-options.md
+```
 
 # ENVIRONMENT VARIABLES
 
-`$FLOX_ENV`
-:   Absolute path to the _install prefix_ of the environment being activated.
-
-    Set by `flox activate` before executing `shell.hook`.
+## `$_FLOX_ACTIVE_ENVIRONMENTS`
+A JSON array containing one object per active environment.
+This is currently an implementation detail and its contents are subject to
+change.
 
-    If multiple environments are indicated, each `shell.hook` will run with
-    its associated `FLOX_ENV` set properly, and the activated environment
-    will have `FLOX_ENV` set to the first environment indicated on the CLI.
-
-    This variable may be used to set other environment variables such as
-    `MANPATH`, `PKG_CONFIG_PATH`, `PYTHONPATH`, etc so that relevant tooling
-    will search these directories to locate files and resources from
-    the environment.
-
-    **N.B.** the default shell hook for newly-created environments will
-    source the `$FLOX_ENV/etc/profile` file at activation if it exists.
-    This behavior can be viewed/modified with `flox edit`.
-
-## Language packs _(**experimental**)_
-
-Language packs help you develop with flox the way **you** work, making it
-possible to install and use compilers, interpreters, libraries and modules
-in much the way you would on any other operating system.
-
-Language packs are activated by way of `$FLOX_ENV/etc/profile` as described
-above, and the `flox.etc-profiles` package provides a version of this script
-along with "language packs" providing environment variables and hooks that
-support developing in a variety of languages.
+# EXAMPLES:
 
-Install a bundle of all language packs with the command:
+Activate an environment stored in the current directory:
```

Comment:
suggestion: use subsections?

Unless that messes up formatting siginficantly, i guess this is providing the right structure

```suggestion
## Activate an environment stored in the current directory:
```

Reply:
The formatting in the terminal is a little weird with subsections (code blocks indented two levels). I also think it looks fine in the rendered HTML without subsections.